src/lib/components/prompt/PromptSuggestions.svelte

```diff
@@ -49,25 +56,26 @@
 		</button>
 	</div>
 	<div class="flex flex-col gap-4 overflow-auto">
```

Comment:
I'm on the plus plan and getting a paywall every time I try to save

Reply:
can't really test this now I'm not on plus and I don't have the correct stripe key